sympy/physics/quantum/tests/test_grover.py

```diff
@@ -48,7 +48,7 @@ def test_OracleGate():
 def test_WGate():
     nqubits = 2
     basis_states = superposition_basis(nqubits)
-    assert qapply(WGate(nqubits)*basis_states) == basis_states
+    assert qapply(WGate(nqubits)*basis_states).simplify().expand() == basis_states
```

Comment:
`simplify` should be avoided in tests. It consists of series of different kinds of simplifications most of which have no effect. Only the effective one should be applied.

Reply:
I guess I am missing something here. If the kets are symbols they would "simplify" automatically. Now they won't.